libraries/AP_Logger/LogStructure.h

```diff
@@ -327,17 +327,68 @@ struct PACKED log_Optflow {
 struct PACKED log_POWR {
     LOG_PACKET_HEADER;
     uint64_t time_us;
+#ifdef ANALOG_VCC_5V_PIN
     float Vcc;
+#endif
+#if HAL_HAVE_SERVO_VOLTAGE
     float Vservo;
+#endif
     uint16_t flags;
     uint16_t accumulated_flags;
     uint8_t safety_and_arm;
+#if HAL_WITH_MCU_MONITORING
```

Comment:
having a separate MCU msg would likely give more flash savings and would fit our current structure